app/analyse/analyser.py

```diff
@@ -97,7 +126,8 @@ def start_analyse(repo, access_token):
             },
         )
         repo_forks_list = res.json()
-        localfile_tool.write_to_file(forks_list_path, repo_forks_list)
+        active_forks = get_active_forks(repo,access_token )
```

Comment:
active_forks should also be passed in project_updater.start_update()

Reply:
Changed project_updater.start_update() call to use active_forks